_posts/2022-03-01-stork-load-balancing.adoc

```diff
@@ -0,0 +1,231 @@
+Thus, while the blue and yellow services still have inflight calls, the green service is back to 0 and selected.
+
+This strategy is convenient when you have services with an API having different response times. 
+For example, you can have endpoints answering quickly and endpoints doing a lot more work, thus taking more time. 
+
+This strategy has one drawback. 
+If you have many service instances, you need to iterate over the whole set to find the one to pick. 
+
+== The power of two random choices
+
+This strategy extends the _least-request_ one and addresses the drawback mentioned above. 
+Instead of iterating over the whole list of instances, it picks two instances randomly and selects the one with the fewer inflight requests.
+
+To use this strategy, you need the following dependency:
+
+[source, xml]
+----
+<dependency>
+ <groupId>io.smallrye.stork</groupId>
+ <artifactId>stork-load-balancer-power-of-two-choices</artifactId>
+</dependency>
+----
+
+You also need to configure the load balancer for the service:
+
+[source, properties]
+----
+stork.my-service.load-balancer=power-of-two-choices
+----
+
+The results are similar to the _least-request_ as we have only three instances:
+
+image::/assets/images/posts/stork-load-balancing/power-of-two-choices.png[align="center"]
+
+However, this strategy is less expensive when you have many instances and when the iteration takes too much time. 
+In the worst case, it would pick the two most busy instances and select the one with fewer requests.
+
+== Least response time
+
+The least-response time keeps track of the response time and failures. 
+Thus, it selects the fastest instances. 
+That strategy handles failures by adding a penalty to failing instances.
+
+To use this strategy, you need the following dependency:
+
+[source, xml]
+----
+<dependency>
+ <groupId>io.smallrye.stork</groupId>
+ <artifactId>stork-load-balancer-least-response-time</artifactId>
+</dependency>
+----
+
+You also need to configure the load-balancer for the service:
+
+[source, properties]
+----
+stork.my-service.load-balancer=least-response-time
+----
+
+This strategy selects the green instance more often. 
+However, due to the chance of failure, it will also pick the other ones once in a while.
+
+image::/assets/images/posts/stork-load-balancing/least-response-time.png[align="center"]
+
+== Summary
+
+This blog post has presented the load balancing strategies offered by Stork. 
+Strategies like `random` or `round-robin` allow dispatching the load between instances.
+The `least-request`, `power-of-two-choices`, and `least-response-time` strategies monitor the calls to select the less loaded or fastest instance.
+
+If these strategies do not fit your requirements, that's not a problem because http://smallrye.io/smallrye-stork/1.0.0/custom-load-balancer/[you can implement your own] and craft the perfect selection algorithm.
```

Comment:
```suggestion
If these strategies do not fit your requirements, that's not a problem because http://smallrye.io/smallrye-stork/1.1.0/custom-load-balancer/[you can implement your own] and craft the perfect selection algorithm.
```